automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -230,7 +230,15 @@ trait Orchestration extends FireCloudClient with LazyLogging {
       parseResponse(getRequest(apiUrl(s"api/methods/$ns/$name/$snapshotId/permissions")))
     }
   }
-
+  /*
+   *  NIH requests
+  */
+   object NIH {
+    def addUserToNIH(jwt: String)(implicit token: AuthToken): Unit = {
```

Comment:
The jwt refers to the JSON web token to link a user to our mock NIH. This parameter is the JWT from NIH login.

Reply:
Ahh, I think I get it. It's a single token, not per-user information. The user added is the one the auth token is associated with. So this is much like the X-FireCloud for Thurloe then?

I'm going suggest then that this be pulled out of config here in `Orchestration` rather than it being a parameter to this function.

Reply:
i think it's still useful to have this function take a jwt. different jwts may be associated with different users on different whitelists.

i.e. we may want a test scenario where a user has a JWT that will get them into the TARGET group but not the TCGA group, or vice versa. or a valid JWT that resolves to an NIH username that isn't on any of our whitelists